test/test_batch_process_endpoints.py

```
@@ -468,6 +468,26 @@ def test_get_users_eligible_for_fist_notification_with_no_result(self):
         self.assertNotIn(self.user_1, users)
         self.assertNotIn(self.user_3, users)
 
+    @patch("ras_rm_auth_service.batch_process_endpoints.transactional_session")
+    def test_get_users_eligible_for_first_notification_with_database_disconnect(self, session_mock):
+        # Given:
+        self.batch_setup()
+        session_mock.side_effect = SQLAlchemyError()
+
+        # When:
+        _datetime_24_months_ago = datetime.utcnow() - timedelta(days=750)
+        criteria = {"last_login_date": _datetime_24_months_ago}
+        criteria_one = {"account_creation_date": _datetime_24_months_ago}
+        self.update_test_data(self.user_0, criteria)
+        self.update_test_data(self.user_2, criteria)
+        self.update_test_data(self.user_1, criteria_one)
+        self.update_test_data(self.user_3, criteria_one)
+        response = self.client.get("/api/batch/account/users/eligible-for-first-notification", headers=self.headers)
```

I haven't dug into the code, but can you get away without all of this, we just want to raise an error

---

Removed and still works